config/schema/graphql.schema.yml

```diff
@@ -72,6 +72,11 @@ graphql.settings:
   label: "GraphQL Settings"
   mapping:
     # @todo Remove in GraphQL 5.
+    dataproducer_allow_current_user_caching:
```

Comment:
I don't think we need this setting. We can add caching, as long as we achieve the same behavior this will not be a compatibility break.